style.css

```diff
@@ -394,3 +395,118 @@ p {
   list-style-type: none;
   text-decoration: none;
 }
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  text-align: center;
+}
+
+.form-section > h2 {
+  margin: 120px 24px 73px 24px;
+  position: static;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  list-style: none;
+}
+
+.input1 {
+  margin: 0 40px 0 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #979493;
+  display: flex;
+}
+
+.email {
+  margin-top: 25px;
+}
+
+.separator1 > div {
+  border: 1px solid #dbd8d7;
+  margin: 14px 24px 24px 24px;
+}
+
+.input1 > input {
+  padding-right: 100px;
+  border: none;
+  width: 295px;
+}
+
+.input2 {
+  margin: 0 24px 24px 24px;
+}
+
+.text-area {
+  background-color: #fbf8f7;
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 12px;
+  width: 303px;
+  height: 90px;
+}
+
+.form-button {
+  text-align: left;
+  margin-left: 112px;
+}
+
+.submit {
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 12px;
+}
+
+.form-button :hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
```

Comment:
Button interactions such as hover, active (when it's pressed), disabled and enable (default state) should be styled as shown in the Figma template.
Please implement the missing states for the links/buttons in your app (at least for the pressed state / active state).
Here is a very useful link to study different states for links/buttons: [Styling links](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links)

Figma design:
![image](https://user-images.githubusercontent.com/83596135/160722459-30ff915e-8929-4d9d-9ed2-b94a97cac0b8.png)